client/lib/plans/constants.js

```diff
@@ -263,7 +263,6 @@ export const PLANS_LIST = {
 			FEATURE_WORDADS_INSTANT,
 			FEATURE_VIDEO_UPLOADS_JETPACK_PREMIUM,
 			FEATURE_MALWARE_SCANNING_DAILY,
-			FEATURE_UNLIMITED_PREMIUM_THEMES,
```

Comment:
Thanks for having a look @ockham It's been decided that themes will be available only for professional plans. What I removed here was the feature for premium plans. The ones for professional and monthly professional is still there.